src/styles/books.scss

```diff
@@ -49,6 +49,11 @@
       display: flex;
```

Comment:
- [ ] `required`
- [ ] Amazing job so far! 👏 
- [ ] However,  I suggest you stick to the design provided because that's the requirement.
- [ ] Please see the screenshots below and make adjustments according to the U.I provide

`The U.I Design provide`
![image](https://user-images.githubusercontent.com/66230306/214138325-74bebb36-b433-426f-86bc-dbead94a7af1.png)

`And your design`
![image](https://user-images.githubusercontent.com/66230306/214138604-0c000292-5d2f-4406-9c9a-71169ee45ab4.png)

Reply:
Thanks for your review, do I have to remove ``Store`` , ``Contacts`` and ``Footer``?

Reply:
Thanks for your review, do I have to remove Store , Contacts and Footer?
@mwanawabangona

Reply:
@belaymit I think if it is not on the Provided design it is wise to ignore it, maybe you can add new features after you have completed this milestone. 🥂 
Happy coding!! 😉